src/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -333,4 +333,7 @@
   <data name="SerializerPropertyNameNull" xml:space="preserve">
     <value>The JSON property name for '{0}.{1}' cannot be null.</value>
   </data>
+  <data name="DeserializeDuplicateKey" xml:space="preserve">
+    <value>An item with the same key has already been added. Key: {0}.</value>
```

Comment:
**Question/nit:** Should we map this exception message closer to what the user passed in/did? The way it is written atm may not be very useful to the caller since it seems more about implementation detail (around adding to a dictionary) than point to an issue with the input. Presumably, this occurs when a JSON payload contains conflicting property names at a particular level/depth, correct? Maybe we should use those terms instead of item/key.

Reply:
Yes. I copied the dictionary message and wasn't sure if a more context-sensitive message would be better, or if we'll have other cases where we want to re-use a more generic message.

However this is probably more useful: "An item with the same property name '{0}' has already been added to the dictionary."

Reply:
Agreed. I like that better. However, is it useful for the user to know that we failed "adding to a dictionary" or can we remove that part?